devel/gdub/Portfile

```diff
@@ -0,0 +1,37 @@
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        dougborg gdub 0.1.0 v
+categories          devel java groovy
+platforms           darwin
+supported_archs     noarch
+maintainers         @breun openmaintainer
+license             MIT
+
+description         A gradlew / gradle wrapper
+long_description    gdub (gw on the command line) is a gradle / gradlew wrapper. \
+                    Not to be confused with the Gradle Wrapper, gw invokes ./gradlew on \
+                    projects where one is configured, and falls back to use the gradle \
+                    from the \$PATH if a wrapper is not available. Also, gw is 66% shorter \
+                    to type than gradle and 78% shorter to type than ./gradlew.
+
+homepage            http://www.gdub.rocks/
+
+checksums           rmd160  fa612e31fd2f32e429225338ff5361184b82b318 \
+                    sha256  76d9a81ba295cf8a4024bde845cd8e36bb01e3465ed09221fb69d7627150afbc
+
+use_configure       no
+build {}
+
+pre-destroot {
+    delete ${worksrcpath}/install
```

Comment:
Is there a reason to delete this? It does not look like this path would even be used in the destroot phase at all.